src/pybind/mgr/dashboard/frontend/src/app/shared/services/auth-storage.service.ts

```diff
@@ -31,7 +36,19 @@ export class AuthStorageService {
     return localStorage.getItem('dashboard_username');
   }
 
+  updateNeedChangePassword(need_change_password: boolean){
+    localStorage.setItem('need_change_password', String(need_change_password));
+  }
+  
+  ForceChangingPasswordAfterFirstLog() {
+    if(localStorage.getItem('need_change_password') == 'false'){ 
+        return;
+      }
+    return this.router.navigate(['/user-management/users/edit/'+ this.getUsername()]);;
```

Comment:
I'm curious here: if the user doesn't (want to) change their password, and manually reset `LocalStorage`, would this policy be enforced by the back-end as well, or it's just a one-time thing (at login step)?